101-webapp-with-golang/azuredeploy.json

```diff
@@ -0,0 +1,129 @@
+            }
+
+        },
+        "sku": {
+            "type": "string",
+            "allowedValues": [
+                "Free",
+                "Shared",
+                "Basic",
+                "Standard"
+            ],
+            "defaultValue": "Free",
+            "metadata": {
+                "description": "SKU value"
+            }
+        },
+        "workerSize": {
+            "type": "string",
+            "allowedValues": [
+                "0",
+                "1",
+                "2"
+            ],
+            "defaultValue": "0",
+            "metadata": {
+                "description": "Worker Size( 0=Small, 1=Medium, 2=Large )"
+            }
+        },
+        "use32bitWorkerProcess": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Enable 32bit worker process or 64bit, 64bit is not available if you choose free hosting plan."
+            }
+        },
+        "enableAlwaysOn": {
+            "type": "bool",
+            "defaultValue": false,
+            "metadata": {
+                "description": "Enable/Disable always on. If you choose free for hosting plan, this should be set to false as always on is not supported for free plan"
+            }
+        }
+    },
+    "variables": {
+        "goSite": "[parameters('siteName')]"
+    },
+    "resources": [
+        {
+            "apiVersion": "2015-04-01",
+            "name": "[parameters('appServicePlanName')]",
+            "type": "Microsoft.Web/serverfarms",
+            "location": "[parameters('siteLocation')]",
+            "properties": {
+                "name": "[parameters('appServicePlanName')]",
+                "sku": "[parameters('sku')]",
+                "workerSize": "[parameters('workerSize')]",
+                "numberOfWorkers": 1
+            }
+        },
+        {
+            "apiVersion": "2015-06-01",
+            "name": "[variables('goSite')]",
+            "type": "Microsoft.Web/sites",
+            "location": "[parameters('siteLocation')]",
+            "dependsOn": [
+                "[resourceId('Microsoft.Web/serverfarms', parameters('appServicePlanName'))]",
+            ],
+            "properties": {
+                "name": "[variables('goSite')]",
+                "serverFarmId": "[parameters('appServicePlanName')]"
+            },
+            "resources": [
+                {
+                    "apiVersion": "2015-06-01",
+                    "name": "web",
+                    "type": "config",
+                    "dependsOn": [
+                        "[resourceId('Microsoft.Web/Sites', variables('goSite'))]"
+                    ],
+                    "properties": {
+                        "phpVersion": "off",
+                        "netFrameworkVersion": "v4.5",
+                        "use32BitWorkerProcess": "[parameters('use32bitWorkerProcess')", /* 64-bit platform */
```

Comment:
Can you remove the comments? the CLI will fail while trying to deploy this template due to comments